game/scripts/npc/abilities/chaos_knight_phantasm.txt

```diff
@@ -41,7 +41,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "images_count"                                    "3"
+        "images_count"                                    "4" //OAA
```

Comment:
Remove all of your CK changes. Let Darkonius CK changes go through first the see how the hero is.

Reply:
i will talk to darkonius